atrium-api/src/types/string.rs

```diff
@@ -436,6 +452,27 @@ impl Tid {
         }
     }
 
+    /// Construct a new timestamp with the specified clock ID.
+    ///
+    /// Clock IDs 0-31 can be used as an ad-hoc clock ID if you are not concerned
+    /// with this parameter.
+    pub fn from_datetime(cid: u32, time: chrono::DateTime<chrono::Utc>) -> Self {
```

Comment:
The parameter name `cid` is easily confused with the `Cid` of `ipld_core`, so I think you should use the name `clock_identifier` or `clock_id` instead of omitting it here.